connectors/google/google-base/src/main/java/io/camunda/google/supplier/util/GoogleServiceSupplierUtil.java

```diff
@@ -18,7 +24,22 @@ public final class GoogleServiceSupplierUtil {
   private GoogleServiceSupplierUtil() {}
 
   public static HttpCredentialsAdapter getHttpHttpCredentialsAdapter(final Authentication auth) {
-    return new HttpCredentialsAdapter(auth.fetchCredentials());
+    Credentials creds = null;
+    if (auth.authType() == AuthenticationType.BEARER) {
+      AccessToken accessToken = new AccessToken(auth.bearerToken(), null);
+      creds = new GoogleCredentials(accessToken).createScoped(DriveScopes.DRIVE);
```

Comment:
Does this only apply for drive?

Reply:
Yes. Probably, it was incorrectly splitted to Drive/Docs and Spreadsheets. I would prefer not to take risk refactoring it few hours before code freeze. Maybe, I will take a look at after codefreeze, since it's not critical IMO. WDYT?